ngraph/core/reference/include/ngraph/runtime/reference/prior_box.hpp

```diff
@@ -107,8 +107,11 @@ namespace ngraph
                     step_y = step;
                 }
 
-                auto calculate_data = [&dst_data, &IWI, &IHI, &idx](
-                    float center_x, float center_y, float box_width, float box_height, bool clip) {
+                auto calculate_data = [&dst_data, &IWI, &IHI, &idx](float center_x,
+                                                                    float center_y,
+                                                                    float box_width,
+                                                                    float box_height,
+                                                                    bool clip) {
```

Comment:
this is inconsistent with the other cases when all of the parameters were moved to the next line.
example:
```
foo(int a,
    int b,
    int c)
```
was changed to:
```
foo(
  int a,
  int b,
  int c)
```

Reply:
This is a result of auto-formatting, so there has to be kind of rule, but not always configurable.